third_party/com_github_ethereum_c_kzg_4844.patch

```diff
@@ -36,3 +19,28 @@
      ],
      importpath = "github.com/ethereum/c-kzg-4844/bindings/go",
      visibility = ["//visibility:public"],
+diff --git a/bindings/go/blst_headers/BUILD.bazel b/bindings/go/blst_headers/BUILD.bazel
+new file mode 100644
+index 0000000..82cad2c
+--- /dev/null
```

Comment:
what's this?

Reply:
Not sure. This was created by `git diff > /path/to/third_party/com_github_ethereum_c_kzg_4844.patch`

Reply:
It means a new file basically